code-analysis/crates/codegen/parser/src/chumsky/boilerplate.rs

```diff
@@ -712,3 +712,70 @@ pub fn parse_macros() -> TokenStream {
         }
     )
 }
+
+pub fn error_renderer() -> TokenStream {
+    quote!(
+        fn render_error_report(error: &ErrorType, with_color: bool) -> Report<SpanType> {
```

Comment:
II agree - it is private because the public api is only ParserOutput::errors_as_strings(). Currently it requires ariadne in the target crate, but I guess we could make that a conditional feature.

Reply:
Sounds good. Thanks.

> but I guess we could make that a conditional feature.

No need, since it is an temporary measure anyways. The current code is fine since it is not part of the public API.